tests_config/run_spec.js

```diff
@@ -56,8 +56,11 @@ const isTestDirectory = (dirname, name) =>
   dirname.startsWith(path.join(__dirname, "../tests", name));
 
 global.run_spec = (fixtures, parsers, options) => {
-  // Can't test plugins on standalone
+  // Can't test plugins on standalone, a empty test to pass
```

Comment:
You can test them if you pass them as objects, not strings.

Reply:
I've tried this before... I forgot. Can't work, maybe because there are `function`s?

Reply:
Okay. Let's keep it this way. I might look at this later.

Reply:
Are you talking about `a empty test to pass`? This not work too, snapshots don't match.

I ignored it in `jest.config.js` for now.

Reply:
No, about passing the plugin as an object. It should work. If it doesn't, something isn't right.